test/markup/markdown/bold_italics.txt

```diff
@@ -6,13 +6,19 @@ __Bold__
 __Bold *then italic*__
 **Bold _then italic_**
 **Bold and <br/>**
-**_[this](https://google.com)_**
+__*[this](https://google.com)*__
 
 > quoted **Bold _then italic_**
-> _[this](https://google.com)_
+> *[this](https://google.com)*
```

Comment:
Ditto.

Reply:
As above, the quoted rule has matched the `> ` immediately before the underscore, preventing the underscore rule from matching the space.

Reply:
This could I think be resolved by simple changing block quote to use a look-ahead for the spaces:

```
  const BLOCKQUOTE = {
    className: 'quote',
    begin: /^>(?=\s+)/,
    contains: inline(),
    end: '$'
  };
```